examples/basic/apps/docs/app/page.module.css

```diff
@@ -214,6 +214,7 @@
   position: absolute;
   min-width: 614px;
   min-height: 614px;
+  pointer-events: none;
```

Comment:
Anywhere you're seeing me do `pointer-events: none` is me working around that the background imagery (the gradient image, the circles image) is sitting on top of nearly the entire page, making it so you can't click the elements underneath.

I could do the work to actually fix this but it's more important to me that we have a great example of RSCs rather than a perfect HTML template (that 99% of folks are deleting anyway).